changelogs/fragments/4506-sudo-in-nmap-inv-plugin.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- nmap inventory plugin - add sudo option in plugin in order to execute the plugin with a non root user with sudo right on nmap executable (https://github.com/ansible-collections/community.general/pull/4506).
```

Comment:
How about:
```suggestion
- nmap inventory plugin - add ``sudo`` option in plugin in order to execute ``sudo nmap`` so that ``nmap`` runs with elevated privileges (https://github.com/ansible-collections/community.general/pull/4506).
```